gradle.properties

```diff
@@ -1,9 +1,10 @@
 version=5.4-SNAPSHOT
 publishUrl=file:../marklogic-data-hub/releases
 systemProp.file.encoding=utf-8
-org.gradle.jvmargs='-Dfile.encoding=UTF-8'
+org.gradle.jvmargs='-Dfile.encoding=UTF-8' '-Dhttps.protocols=TLSv1,TLSv1.1,TLSv1.2'
```

Comment:
What's the purpose of these? I think a comment should be added above this property to explain it's being set in this fashion.

Reply:
Yeah Sure @rjrudin. I'm still testing this.
Workaround to a bug in java 11.
https://stackoverflow.com/questions/52574050/javax-net-ssl-sslexception-no-psk-available-unable-to-resume

Reply:
Build passed all 15times. and added the comment.